src/ser/value.rs

```diff
@@ -51,8 +51,8 @@ impl<'key, 'target, Target> Serializer
     type StructState = ();
     type StructVariantState = ();
 
-    fn serialize_bool(&mut self, _v: bool) -> Result<(), Error> {
-        Err(Error::unsupported_value())
+    fn serialize_bool(&mut self, v: bool) -> Result<(), Error> {
+        self.append_pair(&v.to_string())
```

Comment:
Just do:

```rust
self.append_pair(if v { "true" } else { "false" })
```

This will avoid allocating a new string any time a boolean value needs to be serialised.